frontend/start-dev.js

```diff
@@ -4,10 +4,15 @@ const fs = require("fs");
 const PROXY = 'proxy';
 const COMMON = '@rotki/common'
 const ROTKI = 'rotki'
+const BACKEND = 'backend';
+
+const scriptArgs = process.argv
+const noElectron = scriptArgs.indexOf('--web') >= 0;
```

Comment:
lol, is this the way of doing cli args in node?

Reply:
the default is just an array with all the args, including node and the script.

For my purposes, I just need to check if the --web flag exists or not so no need to go fancy.